src/list_function.py

```diff
@@ -51,13 +51,23 @@ def item_filter(item):
 def handler(event, context):
     pagination_token = (event.get("queryStringParameters") or {}).get("NextToken")
 
+    pagination_token_encryption_context = {
+        # Ensure that a token issued for one user can't be used by another user.
+        # Customize this to use the appropriate identity field for your authorizer.
+        "user": ((event.get("requestContext") or {}).get("identity") or {}).get("user") or "",
```

Comment:
I'd wrap this `get_user(event)` function up in something like `common/event_utils.py`